templates/generic/digest.html

```diff
@@ -0,0 +1,89 @@
+{{period}} github digest ({{#topic}}{{.}}{{/topic}}{{^topic}}{{#repos}}{{shortname}}{{^last}}, {{/last}}{{/repos}}{{/topic}})
+
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>Weekly digest</title>
+<style>
+body { font-family: Gotham, "Helvetica Neue", Helvetica, Arial, sans-serif; }
+h2 { margin-top: 3em; }
+h3 { margin-bottom:0; margin-top: 2em; font-size: 1.4em; }
+a { color: #e6ad49; text-decoration: none; }
+li { margin-bottom: .2em; }
+.new { color: red; }
+</style>
+</head>
+
+<body>
+<h1>Date here</h1>
+<p>{{#filtered}}Events filtered by label {{#labels}}"{{name}}"{{^last}}, {{/last}}{{/labels}}</p>
+{{/filtered}}
+
+{{#activeissues}}<h2>Issues</h2>
+
+{{#activeissuerepos}}
+<h3>{{name}} (+{{newissues.count}}/-{{closedissues.count}}/💬{{issuecommentscount}})</h3>
+{{#newissues.count}}  <p class="new">{{.}} issues created:</p>
+{{#newissues.list}}
+  <ul>
+  <li><a href="{{{payload.issue.html_url}}}">{{{payload.issue.title}}}</a> (by {{actor.display_login}})</li>
```

Comment:
Are you sure?  I thought about that, but didn't think anything inside {{ }} needed html escaping.